Doberman/AlarmMonitor.py

```diff
@@ -113,25 +99,23 @@ def send_email(self, toaddr, subject, message, level, pipeline, cc=None, bcc=Non
             # add manual silence options
             message += '<br><br>To silence the pipeline for longer, click one of the following links:<ul>'
             for silence_for, text in zip((15, 60, 360), ('15 minutes', '1 hour', '6 hours')):
-                if silence_for > int(silence_duration/60):
+                if silence_for > int(silence_duration / 60):
                     message += f'<li><a href="{website_url}/pipeline?pipeline={pipeline}&silence={silence_for}">' \
                                f'{text}</a></li> '
             message += '</ul>'
-        if add_signature:
-            message += f'<hr>Message created on {now} by Doberman slow control.'
-        body = str(message)
-        msg.attach(MIMEText(body, 'html'))
+        message += f'<hr>Message created on {now} by Doberman slow control.'
+        msg.attach(MIMEText(message, 'html'))
         # Connect and send
+        self.logger.warning(f'Sending e-mail to {len(addresses)} recipient{"s" if len(addresses)>1 else ""}')
```

Comment:
Why is this a warning? In the context its worth a warning, but wouldn't it be more an Info, if we reserve warnings for hickups?

Reply:
Just so it gets logged to the database/ is visible on the website when there is an alarm sent to people.